content/security/index.adoc

```diff
@@ -4,48 +4,60 @@ title: "Jenkins Security"
+This project is configured in such a way that only the reporter and the security team can see the details.
+By restricting access to this potentially sensitive information we can work on a fix and deliver it before the method of attack becomes well-known.
+
+To show our appreciation for your help, we offer link:https://wiki.jenkins-ci.org/display/JENKINS/Rewards+for+reporting+security+issues[a small reward] for your report.
+
+== Jenkins Security Team
+
+The Jenkins Security Team is a group of volunteers lead by the link:https://wiki.jenkins-ci.org/display/JENKINS/Team+Leads[Jenkins Security Officer] who triage and fix security vulnerabilities.
+
+Membership is generally open to anyone with a history of positive contributions to the Jenkins project and subject to approval by the Jenkins Security Officer.
+If you want to contribute, be sure to have link:https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/[two-factor authentication enabled on GitHub], link:https://github.com/jenkinsci/infra-cla/[a CLA] in place, a link:https://freenode.net/faq.shtml#userregistration[registered nickname on IRC].
+Then, just contact us on the link:/content/mailing-lists[developers mailing list] with your GitHub, Freenode, and Jenkins community user names.
+
+Security vulnerabilities in core are typically resolved by members of this team, while vulnerabilities in plugins are generally resolved in collaboration with the plugin maintainer(s).
+To that end, plugin maintainers are granted access to issues involving their plugin in our issue tracker, and we create private repositories link:https://github.com/jenkinsci-cert/[in the +jenkinsci-cert+ GitHub organization] for collaboration and PR review.
+
+== Vulnerabilities in Plugins
 
-== Reporting vulnerabilities
+We strive to fix all security vulnerabilities in Jenkins and plugins in a timely manner.
+However, the structure of the Jenkins project and the number of plugins make this impossible:
 
-If you find a vulnerability in Jenkins, please report it in the issue tracker
-under the link:https://issues.jenkins-ci.org/browse/SECURITY[SECURITY project].
-This project is configured in such a way that only the reporter and the core
-Jenkins developers can see the details.
+- Plugins are often independently maintained by developers we may have no direct contact with, or may not have an active maintainer at all.
+- The number of plugins in existence exceeds the security team's capacity to fix them.
 
-By restricting the access to the potential sensitive information, we can work
-on the problem and deliver the fix before the method of attack becomes
-well-known.
+In case of a plugin vulnerability, we try to contact the plugin maintainer(s) to inform them of it.
+If they decline (or otherwise fail) to fix the vulnerability, or don't respond in a timely manner, and the security team doesn't have the capacity to fix it, we follow the process outlined below in the interest of our users:
```

Comment:
I'd rather not unnecessarily restrict us here by making up a process we may not be able to follow. Do we need a general "exceptional situations" clause here? Wasn't intended to be waterproof, just general FYI.